go.mod

```diff
@@ -3,14 +3,15 @@ module github.com/theforeman/ygg_worker
 go 1.19
 
 require (
-	git.sr.ht/~spc/go-log v0.1.1
 	github.com/google/uuid v1.6.0
 	github.com/redhatinsights/yggdrasil v0.4.2
 	github.com/redhatinsights/yggdrasil_v0 v0.0.0-20210811162724-41397343c25b
+	github.com/subpop/go-log v0.1.2
 	google.golang.org/grpc v1.58.3
 )
 
 require (
+	git.sr.ht/~spc/go-log v0.1.1 // indirect
```

Comment:
This does not seems as correct change. This "old" go module should not be required indirectly. If it was required indirectly, then it would cause problems anyway. I guess that you also have to update version of yggdrasil, where `git.sr.ht/~spc/go-log v0.1.1` is not required.